src/Arcus.Scripting.ApiManagement/Scripts/Create-AzApiManagementUserAccountsFromConfig.ps1

```diff
@@ -0,0 +1,215 @@
+                        Remove-AzApiManagementUserFromGroup -Context $apimContext -UserId $userId -GroupId $groupId                        
+                        Write-Verbose "The user with id '$userId' has been removed from the group '$groupId'"
+                    }
+                }
+            }
+
+            $linkedSubscriptions = Get-AzApiManagementSubscription -Context $apimContext -UserId $userId
+            if ($linkedSubscriptions.Count -gt 0) {
+                $linkedSubscriptions | ForEach-Object {
+                    $subscriptionId = $_.SubscriptionId
+                    if (($subscriptions | Where-Object {$_.id -eq $subscriptionId}).Count -eq 0) {
+                        Remove-AzApiManagementSubscription -Context $apimContext -SubscriptionId $subscriptionId                        
+                        Write-Verbose "The subscription with id '$subscriptionId' has been removed"
+                    }
+                }
+            }
+        }
+
+        if ($groups.Count -gt 0) {
+            $groups | ForEach-Object {
+                $groupId = $_.id;
+                $groupDisplayName = $_.displayName;
+                $groupDescription = $_.description;
+
+                $group = New-AzApiManagementGroup -Context $apimContext -GroupId $groupId -Name $groupDisplayName -Description $groupDescription
+                Write-Verbose "A group with id '$groupId' and name '$groupDisplayName' has been created or updated"
+                $userToGroup = Add-AzApiManagementUserToGroup -Context $apimContext -GroupId $groupId -UserId $userId
+                Write-Verbose "The user with id '$userId' has been added to the group with id '$groupId'"
+            }
+        }
+
+        if ($subscriptions.Count -gt 0) {
+            $subscriptions | ForEach-Object {
+                $subscriptionId = $_.id;
+                $subscriptionDisplayName = $_.displayName;
+                $subscriptionAllowTracing = $_.allowTracing;
+                $subscriptionScope = $_.scope;
+
+                if ($subscriptionAllowTracing) {
+                    $subscription = New-AzApiManagementSubscription -Context $apimContext -SubscriptionId $subscriptionId -Name $subscriptionDisplayName -Scope $subscriptionScope -UserId $userId -AllowTracing
+                } else {
+                    $subscription = New-AzApiManagementSubscription -Context $apimContext -SubscriptionId $subscriptionId -Name $subscriptionDisplayName -Scope $subscriptionScope -UserId $userId
+                }
+                Write-Verbose "A subscription for user '$userId' with id '$subscriptionId' and name '$subscriptionDisplayName' for scope '$subscriptionScope' has been created or updated"
+            }
+        }
+
+        Write-Host "User configuration has successfully been applied for user with id '$userId' to Azure API Management service '$ServiceName' in resource group '$ResourceGroupName'" -ForegroundColor Green
+    }
+} catch {
+    Write-Host $_
```

Comment:
Should this be `Write-Error` or something? Or would that block the script before we get to the `throw` with the descriptive exception message?

Reply:
I think it needs to be `Write-Host`, I based the new script on this [existing one](https://github.com/arcus-azure/arcus.scripting/blob/main/src/Arcus.Scripting.ApiManagement/Scripts/Create-AzApiManagementUserAccount.ps1) which uses the same setup.

Reply:
Maybe that one is also wrong? 😅 Think it depends on how the script is run, whether it is stopped on an 'error message' or 'exception throw', right?

Reply:
Changed both 👍🏻 
See [`f412b12` (#422)](https://github.com/arcus-azure/arcus.scripting/pull/422/commits/f412b120b0adc427b96f94c768e942b8b3577a90)